packages/runtime/lib/prom-server.js

```diff
@@ -40,6 +49,41 @@ async function startPrometheusServer (runtime, opts) {
     },
   })
 
+  if (opts.readiness !== false) {
+    async function checkReadiness (runtime) {
+      const workers = await runtime.getWorkers()
+
+      for (const worker of Object.values(workers)) {
+        if (worker.status !== 'started') {
+          return false
+        }
+      }
+      return true
+    }
```

Comment:
I think this can be a top level function and not a closure. If so, move it to the top level.